cmd/sidecars/network-passt-binding/domain/configurator.go

```diff
@@ -143,14 +154,19 @@ func (p PasstNetworkConfigurator) generateInterface() (*domainschema.Interface,
 		ifaceTypeUser     = "user"
 		ifaceBackendPasst = "passt"
 	)
+
+	sourceLinkName, err := p.discoverSourceLinkName()
+	if err != nil {
+		return nil, err
+	}
```

Comment:
nit: I would move the link discovery to top of this function, to make if fail faster.